utils/build/build.js

```diff
@@ -244,6 +244,8 @@ steps.push({
 for (const pkg of workspace.packages()) {
   if (!fs.existsSync(path.join(pkg.path, 'src')))
     continue;
+  if (pkg.usesCustomBuildScript)
```

Comment:
Let's make an exception list in this file and not workspace, to avoid surprises in the future.